simapp/export.go

```diff
@@ -11,7 +11,7 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/x/slashing"
 	"github.com/cosmos/cosmos-sdk/x/staking"
-	"github.com/cosmos/cosmos-sdk/x/staking/expected"
+	"github.com/cosmos/cosmos-sdk/x/staking/exported"
```

Comment:
Alt name proposal: `interface` - CC'ing @alexanderbez @rigelrozanski @sabau @jackzampolin

Reply:
yeah `interface` is a golang keyword which is a bit weird to have as a package name (is this even allowed?)

Reply:
@rigelrozanski brings up a good point that `interface` is a reserved keyword...I'm not sure there is a ideal identifier. So perhaps `exported` will suffice.